enterprise/core-edge/src/test/java/org/neo4j/coreedge/scenarios/ClusterShutdownIT.java

```diff
@@ -95,24 +97,19 @@ private void shouldShutdownEvenThoughWaitingForLock0( int clusterSize, int victi
             final AtomicInteger numberOfInstancesReady = new AtomicInteger();
             for ( int i = 0; i < NUMBER_OF_LOCK_ACQUIRERS; i++ )
             {
-                txExecutor.execute( new Runnable()
-                {
-                    @Override
-                    public void run()
+                txExecutor.execute( () -> {
+                    try ( Transaction tx = victimDB.beginTx() )
+                    {
+                        numberOfInstancesReady.incrementAndGet();
+
+                        tx.acquireWriteLock( someNode.get() );
+                        Thread.sleep( LONGER_TIME );
```

Comment:
Oh right, the transaction should be rolled back on shutdown. It think that's exactly what the test is trying to achieve, i.e., that running transactions do not block the shutdown of the database.